tests-integration/tests/http.rs

```diff
@@ -744,6 +744,12 @@ runtime_size = 8
 max_recv_message_size = "512MiB"
 max_send_message_size = "512MiB"
 
+[grpc.tls]
+mode = "disable"
+cert_path = ""
+key_path = ""
+watch = false
```

Comment:
just remove empty/default entries when mode is "disable"

Reply:
It seems we can't remove it. Because it's a  serialize result from `toml-rs/toml` crate serialize.
https://github.com/GreptimeTeam/greptimedb/blob/418090b4647e5e2d9bc248f940f387070c8479a9/src/common/config/src/config.rs#L88-L91